en/log/flight_review.md

```diff
@@ -0,0 +1,331 @@
+
+
+#### Examples: Good Vibration
+
+[QAV-R 5" Racer](https://docs.px4.io/en/frames_multicopter/qav_r_5_kiss_esc_racer.html) frame (excellent vibration).
+
+![Low vibration QAV-R 5 Racer - raw accel. plot](../../assets/flight_log_analysis/flight_review/vibrations_good_accel.png)
+
+DJI F450 frame (good vibration).
+![Low vibration DJI F450 - raw accel. plot](../../assets/flight_log_analysis/flight_review/vibrations_f450_accel.png)
+
+<!-- https://logs.px4.io/plot_app?log=cd88b091-ec89-457c-85f6-e63e4fa0f51d -->
+
+
+#### Examples: Bad Vibration
+
+<span id="raw_acc_s500"></span>
+S500 frame. Borderline vibration levels - a bit high for x and y (which is typical for an S500 airframe).
+This is at the limit where it starts to negatively affect flight performance.
+
+![Borderline vibration S500 x, y - raw accel. plot](../../assets/flight_log_analysis/flight_review/vibrations_s500_accel.png)
+
+
+Vibration too high. Note how the graph of the z-axis overlaps with the x/y-axis graph:
+
+![Vibrations in landing gear - raw accel. plot](../../assets/flight_log_analysis/flight_review/vibrations_landing_gear_accel.png)
+
+
+
+Vibration levels are too high. Note how the graph of the z-axis overlaps with the x/y-axis graph:
+
+![High vibration in raw accel. plot](../../assets/flight_log_analysis/flight_review/vibrations_too_high_accel.png)
+
+
+Very high (unsafe) vibration levels. 
+
+> **Warning** You should not fly with such high vibration levels.
+
+![Exceedingly high vibration in raw accel. plot](../../assets/flight_log_analysis/flight_review/vibrations_exceedingly_high_accel.png)
+
+
+
+### Solutions
+
+Often a source of vibration cannot be identified from a log alone and the vehicle needs to be inspected.
+There can be a combination of multiple sources.
+
+Solutions and steps to reduce vibrations include:
+- Make sure everything is firmly attached on the vehicle (landing gear, GPS mast, etc.)
+- Use balanced propellers.
+- Make sure to use high-quality components for the propellers, motors, ESC and airframe.
+  Each of these components can make a big difference.
+- Use a vibration-isolation method to mount the autopilot.
+- As a *last* measure, adjust the software filters (see [here](https://docs.px4.io/en/config_mc/racer_setup.html#filters)).
+  It is better to reduce the source of vibrations, rather than filtering them out in software.
+
+<!-- TODO: write a separate vibration setup page in more depth, move some of this there and link to it from here -->
+
+
+## Actuator Outputs
+
+The *Actuator Outputs* graph shows the signals that are sent to the individual actuators (motors/servos).
+Generally it is in the range between the minimum and maximum configured PWM values (e.g. from 1000 to 2000).
+
+This is an example for a quadrotor where everything is OK (all of the signals are within the range, approximately overlap each other, and are not too noisy):
```

Comment:
I updated this. Is this a good description of why this is a good plot?

Reply:
:+1: